joblib/test/test_disk.py

```diff
@@ -33,23 +33,27 @@ def test_disk_used(tmpdir_path):
     assert disk_used(cachedir) < target_size + 12
 
 
-@parametrize(['text', 'value', 'exception'],
-             [('80G', 80 * 1024 ** 3, None),
-              ('1.4M', int(1.4 * 1024 ** 2), None),
-              ('120M', 120 * 1024 ** 2, None),
-              ('53K', 53 * 1024, None),
-              ('fooG', None, ValueError)])
-def test_memstr_to_bytes(text, value, exception):
-    if not exception:
-        assert memstr_to_bytes(text) == value
-    else:
-        assert_raises(exception, memstr_to_bytes, text)
+@parametrize(['text', 'value'],
+             [('80G', 80 * 1024 ** 3),
+              ('1.4M', int(1.4 * 1024 ** 2)),
+              ('120M', 120 * 1024 ** 2),
+              ('53K', 53 * 1024)])
+def test_memstr_to_bytes(text, value):
+    assert memstr_to_bytes(text) == value
 
 
-def test_mkdirp(tmpdir_path):
-    mkdirp(os.path.join(tmpdir_path, 'ham'))
-    mkdirp(os.path.join(tmpdir_path, 'ham'))
-    mkdirp(os.path.join(tmpdir_path, 'spam', 'spam'))
+@parametrize(['text', 'exception', 'exc_substr'],
```

Comment:
I mean you are parametrizing over one item so there is no need to parametrize and you can use local variables into a standard test function.